blocks/block-helpers.js

```diff
@@ -17,3 +16,33 @@ export const replaceElementType = (el, type) => {
   [...el.attributes].forEach((attr) => newEl.setAttribute(attr.nodeName, attr.nodeValue));
   return newEl;
 };
+
+/**
+ * Create element with attributes
+ * @param {string} name The tag of the element, ex) <div>
+ * @param {object} attrs The attributes to add to the element, ex) {class: 'class-name'}
+ * @returns el Created Element
+ */
+export function createTag(name, attrs) {
+  const el = document.createElement(name);
+  if (typeof attrs === 'object') {
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [key, value] of Object.entries(attrs)) {
+      el.setAttribute(key, value);
+    }
+  }
+  return el;
+}
```

Comment:
I'd probably move https://github.com/adobe/blog/blob/main/blocks/gnav/gnav-utils.js#L8-L23 to here rather than writing a new function.

Reply:
Good catch, I didn't see that it already existed